### ISSUE TITLE
zip_object error when is called with an empty list 

### ISSUE THREAD

**Issue by 4l1fe** [dgilland/pydash#235]
Hey. I realy appreciate what you've done and am using pydash intensivelly to handle unstructured text from html pages.

Recently, found out that i can't handle an empty list to dict with `zip_object()`, but expect it to be ignored by `zip_object` as it goes the next after the `map` call that return either an empty list `[]` or one with key, value pairs `[['k1', 'some text'], ['k2', 'more text]`

Pls take a look at the error

```python
In [9]: (py_.chain([1,2,3])
   ...: .get('nothing_attr') # emualting handling that returns None
   ...: .map(lambda item: (item,item))
   ...: .zip_object()
   ...: .value())

--------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[9], line 5
      1 (py_.chain([1,2,3])
      2 .get('nothing_attr') # emualting handling that returns None
      3 .map(lambda item: (item,item))
      4 .zip_object()
----> 5 .value())

File ~/.pyenv/versions/3.11.8/envs/pipelines/lib/python3.11/site-packages/pydash/chaining/chaining.py:47, in Chain.value(self)
     40 def value(self) -> ValueT_co:
     41     """
     42     Return current value of the chain operations.
     43
     44     Returns:
     45         Current value of chain operations.
     46     """
---> 47     return self(self._value)

File ~/.pyenv/versions/3.11.8/envs/pipelines/lib/python3.11/site-packages/pydash/chaining/chaining.py:108, in Chain.__call__(self, value)
     97 """
     98 Return result of passing `value` through chained methods.
     99
   (...)
    104     Result of method chain evaluation of `value`.
    105 """
    106 if isinstance(self._value, ChainWrapper):
    107     # pylint: disable=maybe-no-member
--> 108     value = self._value.unwrap(value)
    109 return value

File ~/.pyenv/versions/3.11.8/envs/pipelines/lib/python3.11/site-packages/pydash/chaining/chaining.py:152, in ChainWrapper.unwrap(self, value)
    149 if wrapper._value is not UNSET:
    150     value = wrapper._value
--> 152 return wrapper.method(value, *wrapper.args, **wrapper.kwargs)

File ~/.pyenv/versions/3.11.8/envs/pipelines/lib/python3.11/site-packages/pydash/arrays.py:2699, in zip_object(keys, values)
   2675 """
   2676 Creates a dict composed of lists of keys and values. Pass either a single two-dimensional list,
   2677 i.e. ``[[key1, value1], [key2, value2]]``, or two lists, one of keys and one of corresponding
   (...)
   2695     Removed alias ``object_``.
   2696 """
   2698 if values is None:
-> 2699     keys, values = unzip(keys)
   2701 return dict(zip(keys, values))

ValueError: not enough values to unpack (expected 2, got 0)
```

The function's signature i'm using zip_object with is
```python
In [13]: py_.zip_object?
Signature: py_.zip_object(keys, values=None)
Docstring:
Creates a dict composed of lists of keys and values. Pass either a single two-dimensional list,
i.e. ``[[key1, value1], [key2, value2]]``, or two lists, one of keys and one of corresponding
values.

```

If it is a valid behavior that breaks a whole functional chain? Shoudn't it to be mentioned in the docs? or it is an actual error?

Thx 🙏🏻

**Comment by dgilland** [comment]
Thanks for reporting! This was indeed a bug. Fixed in v8.0.4: https://pypi.org/project/pydash/8.0.4/

**Comment by 4l1fe** [comment]
> Thanks for reporting! This was indeed a bug. Fixed in v8.0.4: https://pypi.org/project/pydash/8.0.4/

Happy to hear, thx for fixing.
